fisher.fish

```diff
@@ -117,7 +117,7 @@ function $fisher_cmd_name -d "fish plugin manager"
     command mkdir -p {"$fish_path","$fish_config"}/{conf.d,functions,completions} "$fisher_config" "$fisher_cache"
     or return 1
 
-    set -l completions "$fish_config/completions/$fisher_cmd_name.fish"
+    set -l completions "$fish_path/completions/$fisher_cmd_name.fish"
```

Comment:
@externl No, that's different. Fisherman will correctly take care of plugins it install and make sure functions/completions/snippets are available after they are installed regardless of their location.

The thing is, fisher completions should be in $fish_complete_path, otherwise `fisher` completions won't work.